meetings/2024-07-04/minutes.md

```diff
@@ -0,0 +1,16 @@
+# 2024-07-04 :  TSC Agenda
+
+* Welcome
+* [Minutes/actions from previous meeting](https://github.com/pq-code-package/tsc/pull/59/files)
```

Comment:
should this be https://github.com/pq-code-package/tsc/pull/78 ?

Reply:
corrected -- thanks